actionpack/test/dispatch/content_security_policy_test.rb

```diff
@@ -366,8 +378,8 @@ def index
   end
 
   POLICY = ActionDispatch::ContentSecurityPolicy.new do |p|
-    p.default_src -> { :self  }
-    p.script_src  -> { :https }
+    p.default_src -> { :self }
+    p.script_src -> { :https }
```

Comment:
I get that these cleanups might be nice while we're editting the file, some committer or core maybe accept it, but it does go against our policy for not accepting cosmetic changes.